app/src/main/java/accesscollective/uwastudentguild/com/accesscollective/MainActivity.java

```diff
@@ -21,6 +23,9 @@
 public class MainActivity extends AppCompatActivity
         implements NavigationView.OnNavigationItemSelectedListener, OnMapReadyCallback {
 
+    String subject= "Access Collective App feedback";
```

Comment:
I'd probably shift these global variables to be within the onNavigationItemSelected function - it's good practice to [avoid using global variables](http://wiki.c2.com/?GlobalVariablesAreBad)